oxidation/client/src/common/mocks.ts

```diff
@@ -0,0 +1,108 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+import { Storage } from '@ionic/storage';
+
+import { AvailableDevice } from '../plugins/libparsec/definitions';
+
+const MOCK_DEVICES: AvailableDevice[] = [
+  {
+    organizationId: 'Planet Express',
+    humanHandle: 'Dr. John A. Zoidberg',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug1',
+    ty: {tag: 'Password'}
+  },
+  {
+    organizationId: 'Princeton–Plainsboro Hospital',
+    humanHandle: 'Dr. Gregory House',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug2',
+    ty: {tag: 'Password'}
+  },
+  {
+    organizationId: 'Black Mesa',
+    humanHandle: 'Dr. Gordon Freeman',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug3',
+    ty: {tag: 'Password'}
+  },
+  {
+    organizationId: 'OsCorp',
+    humanHandle: 'Dr. Otto G. Octavius',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug4',
+    ty: {tag: 'Password'}
+  },
+  {
+    organizationId: 'Sanctum Sanctorum',
+    humanHandle: 'Dr. Stephen Strange',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug5',
+    ty: {tag: 'Password'}
+  },
+  {
+    organizationId: 'Holmes Consulting',
+    humanHandle: 'Dr John H. Watson',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug6',
+    ty: {tag: 'Password'}
+  },
+  {
+    organizationId: 'Riviera M.D.',
+    humanHandle: 'Dr. Nicholas "Nick" Riviera',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug7',
+    ty: {tag: 'Password'}
+  }
+];
+
+export function getMockDevices(count = -1): AvailableDevice[] {
```

Comment:
count = -1 to getAll is quite ugly, just put count as optional and if not set, so return All